thor/bytes32.go

```diff
@@ -46,9 +45,7 @@ func (b Bytes32) IsZero() bool {
 
 // MarshalJSON implements json.Marshaler.
 func (b Bytes32) MarshalJSON() ([]byte, error) {
-	// Convert Bytes32 to a hexadecimal string.
```

Comment:
@otherview I feel this comment is not necessary since the type is not pointer, so removed.

Reply:
Yeah, fair point. Would it be worth to keep a reference to it being a hex encode value ?